integrations/stitch/query/queries.ts

```diff
@@ -108,29 +108,33 @@ export const IdentityQuery = gql`
 export const AccountHolderQuery = gql`
     query GetAccounts {
         user {
-            bankAccounts {
-                accountHolder {
-                    ... on Individual {
-                        gender
-                        fullName
-                        dateOfBirth
-                        email
-                        familyName
-                        givenName
-                        identifyingDocument {
-                            ... on IdentityDocument {
-                            __typename
-                            country
-                            number
-                            }
-                        }
-                        middleName
-                        nickname
-                        contact {
-                            name
-                            phoneNumber
+        bankAccounts {
+            accountHolder {
+                ... on Individual {
+                    gender
+                    fullName
+                    dateOfBirth
+                    email
+                    familyName
+                    givenName
+                    identifyingDocument {
+                        ... on IdentityDocument {
+                        __typename
+                        country
+                        number
                         }
                     }
+                    middleName
+                    nickname
+                    contact {
+                        name
+                        phoneNumber
+                    }
+                }
```

Comment:
Why the indentation shift? `bankAccounts` is still nested under `user`.

```suggestion
            bankAccounts {
                accountHolder {
                    ... on Individual {
                        gender
                        fullName
                        dateOfBirth
                        email
                        familyName
                        givenName
                        identifyingDocument {
                            ... on IdentityDocument {
                            __typename
                            country
                            number
                            }
                        }
                        middleName
                        nickname
                        contact {
                            name
                            phoneNumber
                        }
                    }
```